src/app/components/Home/Home.jsx

```diff
@@ -15,9 +15,11 @@ const Home = ({ sortBy }) => (
 
     <div className="content-wrapper">
       <div className="nypl-column-three-quarters nypl-column-offset-one">
```

Comment:
it makes a difference:

![image](https://cloud.githubusercontent.com/assets/133020/25913615/654646d4-3589-11e7-88ee-33b7c0028be8.png)